src/test/kotlin/uk/gov/dluhc/printapi/scheduler/ProcessPrintResponsesSchedulerIntegrationTest.kt

```diff
@@ -0,0 +1,49 @@
+package uk.gov.dluhc.printapi.scheduler
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.integration.support.MessageBuilder
+import uk.gov.dluhc.printapi.config.IntegrationTest
+import java.io.File
+
+internal class ProcessPrintResponsesSchedulerIntegrationTest : IntegrationTest() {
+
+    @Autowired
```

Comment:
Rather than autowire into individual integration tests please put this in the base class `IntegrationTest`
(otherwise we'll end up with millions of different spring contexts in test and we'll end up with "Ocado slow" tests! 😢  )